framework/decode/vulkan_replay_dump_resources_common.cpp

```diff
@@ -1135,5 +1103,26 @@ uint32_t FindTransferQueueFamilyIndex(const VulkanDeviceInfo::EnabledQueueFamily
     return index;
 }
 
+bool CullDescriptor(CommandImageSubresourceIterator cmd_subresources_entry,
+                    uint32_t                        desc_set,
+                    uint32_t                        binding,
+                    uint32_t                        array_index,
+                    VkImageSubresourceRange*        subresource_range)
+{
+    const DescriptorTuple tuple                   = DescriptorTuple{ desc_set, binding, array_index };
```

Comment:
I think "tuple" shouldn't be part of the variable and maybe not part of the type.   I think it's a little like calling an array index "index_int".  Is this a fully realized descriptor?  Could it be something like DescriptorSpecifics/desc_specifics or DescriptorLocator/descriptor, or something like that?

Reply:
How about `DescriptorLocation`?

Reply:
Sure!